trillian/ctfe/handlers.go

```diff
@@ -199,8 +199,13 @@ func (a AppHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 type CertValidationOpts struct {
 	// trustedRoots is a pool of certificates that defines the roots the CT log will accept
 	trustedRoots *PEMCertPool
+	// currentTime is the time used for checking a certificate's validity period
+	// against. If it's zero then time.Now() is used.
+	currentTime time.Time
```

Comment:
Other places in the codebase use a `util.TimeSource` to allow test-only injection of fake times -- probably better to be consistent?

Reply:
In this case I made it the same way as in `x509.VerifyOptions`. Not sure which way is better: adding `TimeSource` probably requires more plumbing.

Reply:
Fair enough, stet.